src/main/scala/com/atomist/rug/runtime/js/interop/jsSafeCommittingProxy.scala

```diff
@@ -175,7 +175,10 @@ class jsSafeCommittingProxy(
       nodesAccessedThroughThisFunctionCall.toList match {
         case Nil => throw new RugRuntimeException(name, s"No children or function found for property $name on $node")
         case null :: Nil => null
```

Comment:
First case is an empty list. Second is a list with one null element. I don't remember where this is used, but it's not new.